lib/bootstrap_form/helpers/bootstrap.rb

```diff
@@ -25,7 +25,7 @@ def alert_message(title, options = {})
       def error_summary
         content_tag :ul, class: 'rails-bootstrap-forms-error-summary' do
           object.errors.full_messages.each do |error|
-            concat content_tag(:li, error)
+            concat content_tag(:li, error.html_safe)
```

Comment:
I may be stretching it here (but that's what security is about):

1. Suppose a programmer wants to give good feedback to the user, and so echoes some of the user's (invalid) input back to the user in the error message.
2. A malicious user notices this and crafts invalid input that contains some sort of scripting attack

Can someone with more knowledge comment here, please (@mattbrictson)? Are error messages guaranteed to be html safe/escaped? Would it be sufficient to simply say that the errors array should be populated with html-safe strings if that's what the programmer intends?